minicar/tests/test-sensor-IRInduction.js

```diff
@@ -0,0 +1,70 @@
+console.log("Minicar using IR infrared sensor avoid obstacle");
+
+var gpio = require("gpio");
+var pwm = require("pwm");
+var pins = require("arduino101_pins");
+var steerer = require("Steerer.js");
+var driver = require("Driver.js");
+
+// Minicar init
+var steerPin = pwm.open({ channel: pins.IO3 });
+var forwardPin = pwm.open({ channel: pins.IO6 });
+var reversePin = pwm.open({ channel: pins.IO5 });
+
+steerer.init(steerPin);
+driver.init(forwardPin, reversePin);
```

Comment:
it's better to use 'setPin()' method then set pin in init().

For driver module, suggest to use "setForwadPin()" and "setReversePin()".